packages/@aws-cdk/aws-chatbot/lib/slack-channel-configuration.ts

```diff
@@ -268,6 +268,13 @@ export class SlackChannelConfiguration extends SlackChannelConfigurationBase {
 
   readonly grantPrincipal: iam.IPrincipal;
 
+  /**
+   * The SNS topic that deliver notifications to AWS Chatbot.
+   * @attribute
+   */
+  readonly notificationTopics: sns.ITopic[];
```

Comment:
This should be `private`, right?

Reply:
Done. I included a method to add a notification, but what happens if somebody wants to delete notification? Maybe I should change my approach and provide only this attribute as public. What do you think?

Reply:
Nah, let's leave it as-is. If there's need for removal, we can always add it later.